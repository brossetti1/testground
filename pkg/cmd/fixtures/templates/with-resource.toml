[metadata]
  name = "experiment"

[global]
  plan = "plan"
  case = "case"
  builder = "docker:go"
  runner = "local:docker"

{{ with (load_resource "./resource.toml" ) -}}
[[groups]]
  id = "simple"
  instances = { count = 1 }

  [groups.build]
    selectors = ['{{ .selector }}']

  [groups.build_config]
    build_base_image = 'golang:{{ .go_version }}-buster'
    modfile = "{{ .modfile }}"
{{- end -}}