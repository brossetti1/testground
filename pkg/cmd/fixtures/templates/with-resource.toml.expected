[metadata]
  name = "experiment"

[global]
  plan = "plan"
  case = "case"
  builder = "docker:go"
  runner = "local:docker"

[[groups]]
  id = "simple"
  instances = { count = 1 }

  [groups.build]
    selectors = ['v0.21']

  [groups.build_config]
    build_base_image = 'golang:1.18-buster'
    modfile = "go.v0.21.mod"